quotes/templates/quotes/quotes_list.html

```diff
@@ -1,27 +1,60 @@
 {% extends 'internal/base.html' %}
+{% load static %}
+{% load i18n %}
+{% block header %}
+    <link rel="stylesheet" href="{% static 'quotes/css/quotes.css' %}">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+{% endblock %}
 {% block body_content %}
-  <h2>Pending</h2>
-  <ul>
-    {% for quote in pending %}
-      <li>{{ quote.text }}</li>
-    {% endfor %}
-  </ul>
-  <h2>Quotes</h2>
-  <ul>
-    {% for quote in quotes %}
-      <li>
-        <span class="quote__text">
-          {{ quote.text }}
-        </span> -&nbsp;
-        <span class="quote__sum">
-          {{ quote.sum }}
-        </span>
-        &nbsp;
-        <button class="quote__voteup" data-quote-id="{{ quote.id }}">Vote up</button>
-        <button class="quote__votedown" data-quote-id="{{ quote.id }}">Vote down</button>
-      </li>
-    {% endfor %}
-  </ul>
-  {% load static %}
-  <script src="{% static 'quotes/quotes_list.js' %}"></script>
+    <!--TODO: Clean up classnames in css file and remove unused parts and reformat and make cleaner-->
+    <!--TODO: Shift content more to the left -->
+    <div class="profilepage">
+        <div class="default-container default-top-offset">
+            <div id="left-wrapper" class="default-left-panel profilepage__thumbwrapper text-right">
+                <h3>SITATER</h3>
+                <p><a class="btn btn-primary" href="/internal/quotes/add/">Opprett</a></p>
+                <p>Arkiv</p>
+                <p>Søk</p>
+            </div>
+            <div class="default-right-panel" id="right-wrapper">
+                {% if pending|length != 0 %}
+                    <h2>Ikke godkjente</h2>
+                    <div class="grid-container">
+                    {% for quote in pending %}
+                        <span class="grid-item">
+                {{ quote.text }}<br>
```

Comment:
The indentation in the entire file is off.

Also, personally I prefer indenting html with 2 (spaces) instead of 4.